index.html

```diff
@@ -111,10 +111,13 @@
 
 <br>
 
-<div>
+<div style="height: 30px">
     <pre style="display: inline;" id="safetyFactor" title="Liquidation possible when safety factor < 1.0">New safety factor: ∞</pre>
     <input type="button" id="btnFrob">
 </div>
+<div>
+    <input type="button" id="btnPoke">
```

Comment:
Unrelated to this line, the two buttons have different fonts now. In css the btnFrob is set to monospace. Should keep it the same for bothkeep